src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
```

Comment:
Consider renaming the 'cost' constant to 'DAILY_RENTAL_RATE' to more clearly reflect its purpose as a constant value representing the daily rate for renting a car.